vnext/src/Libraries/Components/View/ViewPropTypes.windows.js

```diff
@@ -412,6 +412,8 @@ type WindowsViewProps = $ReadOnly<{|
   accessibilityPosInSet?: ?number,
   accessibilitySetSize?: ?number,
 
+  accessibilityLandmarkType?: ?string,
```

Comment:
I'm not sure if we are producing typings from this (this looks like Flow?) but in any case we should specify what the "known values" are, so something like: 'value1' | 'value2' | string